inventory/virt/group_vars/all

```diff
@@ -0,0 +1,6 @@
+ansible_ssh_user: root
+openshift_hostname: "{{ ansible_default_ipv4.address }}"
+openshift_public_hostname: "{{ ansible_default_ipv4.address }}"
+openshift_ip: "{{ ansible_default_ipv4.address }}"
+openshift_public_ip: "{{ ansible_default_ipv4.address }}"
+openshift_env: "{{ oo_env }}"
```

Comment:
You shouldn't need to define openshift_hostname, openshift_public_hostname, openshift_ip, or openshift_public_ip, the openshift_facts module should default these correctly.

openshift_env and oo_env vars are no longer used.